### DISCUSSION TITLE
How to render any content type in a theme

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#10874] [Q&A] [answered]
I created content type "Button". In theme layout, "Layout.liquid" I try to display an instance of the "Button" content type like this:
`{% shape "button" %}` but I get an exception saying Shape "button" not found. So I added "Button.liquid" in the views folder of the theme project. It resolves the exception but there still no button. Please help. I am new to Orchard Core CMS and I am stuck. I am also struggling to find the documentation on displaying custom content types. I'd like to to have one button type and then simply create a new button and display it where ever I want in my theme.
[This is how it looks now](https://imgur.com/a/4Fb2HWa)
[This is how it should look](https://imgur.com/a/h1F3mBX)

**Comment by Piedone** [comment, marked as answer]
Some clarification on the basics:

- What you want to render is a content **item**, not a content **type**. A content type describes how its content items should work (like a class), and then you create content items with that type (like objects). You need to create a specific Button content item.
- `{% shape "button" %}` displays a [shape](https://docs.orchardcore.net/en/latest/docs/reference/core/Placement/#shapes) called "button". There's no connection of this to the Button (note the capital B) content type, or its items.

You can use `shape_build_display` to take a content item and build the display shape for it, and then `shape_render` to render it into HTML: https://docs.orchardcore.net/en/latest/docs/reference/modules/Liquid/#shape_build_display
